CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 Change Log
 ==========
 
+## Version 0.22
+
+_2019-12-9_
+
+### Kotlin
+
+ * BREAKING: Safer BackStackScreen construction. (#809)
+ * BREAKING: Pass ContainerHints via setContentWorkflow. (#808)
+ * Add variance to StatelessWorkflow type parameters. (#790)
+ * BREAKING CHANGE: ViewRegistry is now a ContainerHint. (#770)
+ * Make `lifecyclOrNull` public. (#747)
+ * Give Worker.doesSameWorkAs a default implementation that just compares by concrete type. (#746)
+ * BREAKING: Remove deprecated `testRender` API. (#743)
```

Comment:
Also this change is gonna make adoption pretty slow I think – was hoping to convert more internal tests before releasing this version :(

Reply:
Want me to revert it for this release?

Reply:
Actually I just grep'd and there aren't all that many of them in our source, just ten files. I'll take a crack at converting.

Reply:
All internal tests have been converted, this is no longer a blocker.